server/src/main/java/org/opensearch/action/admin/cluster/node/stats/NodeStats.java

```diff
@@ -506,6 +523,9 @@ public void writeTo(StreamOutput out) throws IOException {
         if (out.getVersion().onOrAfter(Version.V_2_12_0)) {
             out.writeOptionalWriteable(admissionControlStats);
         }
+        if (out.getVersion().onOrAfter(Version.V_2_13_0)) {
```

Comment:
This needs to be set to current initially and after the PR is merged, we need to set this to 2.13 version.